queries/ses_bounce_ratio.sql.tftpl

```diff
@@ -0,0 +1,12 @@
+select bounces.date, bounces, send, ((bounces * 1.0)/(send * 1.0) * 100 ) as "% bounces"
+    from
+        ( select
+            date(from_iso8601_timestamp(mail.timestamp)) as date, count(*) as bounces from ses.ses_bounce_logs where  eventtype='Bounce'and bounce.bouncedrecipients[1].diagnosticcode not like '%Amazon SES did not send the message to this address%'
+        GROUP BY date(from_iso8601_timestamp(mail.timestamp)) order by date(from_iso8601_timestamp(mail.timestamp)) desc) bounces
+
+    inner join
+        ( select
+            date(from_iso8601_timestamp(mail.timestamp)) as date, count(*) as send from ses.ses_bounce_logs where  eventtype='Send'
+        GROUP BY date(from_iso8601_timestamp(mail.timestamp)) order by date(from_iso8601_timestamp(mail.timestamp)) desc) send
+    ON
+     bounces.date = send.date
```

Comment:
**question**: this returned only one data point for a single date with a bounce rate above 100%. Seems wrong to me.